staging/src/k8s.io/client-go/tools/auth/exec/exec_test.go

```diff
@@ -46,6 +47,50 @@ func TestLoadExecCredential(t *testing.T) {
 		wantRESTInfo       restInfo
 		wantErrorPrefix    string
 	}{
+		{
+			name: "v1 happy path",
```

Comment:
non happy path tests for v1?

Reply:
https://github.com/kubernetes/kubernetes/pull/102890/commits/94aece4d883762a9739a7f1b588309e38b308a83